### ISSUE TITLE
move OSD-chapter : help !

### ISSUE THREAD

**Issue by OmiChrome** [vc-01/metadata-osd#3]
![image](https://user-images.githubusercontent.com/108952834/193671723-48532814-20fd-48f7-8e49-3851bac745f8.png)
  i hope u can see, the text in centre . that is `chapter` which bother too much when watching . i dont to block them 

i just want to move them somwhere else . as at ` bottom or top  `! 
 🙏  thx for that metadata-osd

**Comment by vc-01** [comment]
I agree that some general styling options would be nice to have and asked for sooner or later (maybe sooner than later, when I'm looking at it). :)

For now though, your only option is to disable the OSD-2 altogether if it's bothering too much. Put the following line in the config file:
`enable_osd_2=no`

**Comment by OmiChrome** [comment]
> I agree that some general styling options would be nice to have and asked for sooner or later (maybe sooner than later, when I'm looking at it). :)
> 
> For now though, your only option is to disable the OSD-2 altogether if it's bothering too much. Put the following line in the config file: `enable_osd_2=no`

:P thx

**Comment by vc-01** [comment]
I will close this issue, only once this is implemented, further discussions may raise as well. Thanks for understanding.

**Comment by vc-01** [comment]
With the lastest changes, it's now possible to style the text a bit.

To your request; to move OSD-2 down to 'subtitle' position add the following to your config file:

```
style_alignment_osd_2=subtitle
```

To make it go to 'toptitle', top padding has to be removed as well (set to zero), so it goes:

```
style_paddingtop_osd_2_textarea_1=0
style_alignment_osd_2=toptitle
```

_Note_: It looks better, at least to my eyes if the text is padded at top thence moved a bit down from centre for some reason.

Other combinations with left/centered/right justifications are possible as well, so it can go e.g.:

```
style_paddingtop_osd_2_textarea_1=0
style_alignment_osd_2=toptitle;right_justified
```

I will release this a bit later after it settles and no obvious bugs pop up.

_Note_: The whole styling approach is based on the same styling regular subtitles can have, so the similarities as 'subtitle' notations etc.

**Comment by OmiChrome** [comment]
`chad` has done the work  🙏
